plan/logical_plan_test.go

```diff
@@ -1171,6 +1171,14 @@ func (s *testPlanSuite) TestValidate(c *C) {
 			sql: "select a from t having sum(avg(a))",
 			err: ErrInvalidGroupFuncUse,
 		},
+		{
+			sql: "select concat(c_str, d_str) from t group by `concat(c_str, d_str)`",
+			err: nil,
+		},
+		{
+			sql: "select concat(c_str, d_str) from t group by `concat(c_str,d_str)`",
```

Comment:
This should not result in an error.

Reply:
this is the result of mysql:
```sql
MySQL(localhost:3306) > select concat(k1, k2) from t group by concat(k1,k2);
+----------------+
| concat(k1, k2) |
+----------------+
| 12             |
+----------------+
1 row in set (0.01 sec)
```

Reply:
mysql> select concat(k1, k2) from t group by `concat(k1,k2)`;
    ERROR 1054 (42S22): Unknown column 'concat(k1,k2)' in 'group statement'

\`concat(k1,k2)\` is different from concat(k1,k2).

Reply:
😂 Got it

Reply:
It'll be better to add the test case `select concat(k1, k2) from t group by concat(k1,k2);` here.

Reply:
@XuHuaiyu TiDB always support this case.😂